lib/measures/tbd/measure.xml

```diff
@@ -2,8 +2,13 @@
   <schema_version>3.0</schema_version>
   <name>tbd_measure</name>
   <uid>8890787b-8c25-4dc8-8641-b6be1b6c2357</uid>
-  <version_id>f60a9eb1-6e4f-45d3-8dc3-7c17b5ac99ac</version_id>
-  <version_modified>20210115T104023Z</version_modified>
+<<<<<<< HEAD
```

Comment:
@rd2 you will want to resolve this merge conflict, I'd probably pick the top one since it is newer

Reply:
Aargh! Checked every file for a merge conflict except this one.